src/midiEditorCommands.cpp

```diff
@@ -1443,17 +1553,18 @@ void postMidiSwitchCCLane(int command) {
 }
 
 void postToggleMidiInputsAsStepInput(int command) {
-	ostringstream s;
-	s << (GetToggleCommandState2(SectionFromUniqueID(MIDI_EDITOR_SECTION),
-		command) ? "enabled" : "disabled");
-	s << " MIDI inputs as step input";
-	outputMessage(s);
+	bool enabled = GetToggleCommandState2(SectionFromUniqueID(MIDI_EDITOR_SECTION), command);
+	if(enabled){
```

Comment:
Your update seems to have gotten lost here somehow...
```suggestion
	if (GetToggleCommandState2(SectionFromUniqueID(MIDI_EDITOR_SECTION), command)) {
```